packages/platform/atoms/availability/AvailabilitySettings.tsx

```diff
@@ -76,12 +76,12 @@ export type CustomClassNames = {
   formClassName?: string;
   timezoneSelectClassName?: string;
   subtitlesClassName?: string;
-  scheduleClassName?: {
-    scheduleContainerClassName?: string;
-    scheduleDayClassName?: string;
-    dayRangesClassName?: string;
-    timeRangesClassName?: string;
-    labelAndSwitchContainerClassName?: string;
+  scheduleClassNames?: {
```

Comment:
Had to rename these since there was naming mismatches happening at the time of spreading